docs/CHANGELOG.rst

```diff
@@ -44,91 +44,97 @@ and anyone can contribute to it.
 It is possible to perform full-scale analyses and contribute to the production
 of CTA performances for different science cases or perform studies that require big amounts of data.
 
+
 Contributors
 ++++++++++++
 
-- Michele Peresano (`@HealthyPear`_)
+@HealthyPear and @kosack
```

Comment:
here you can be more explicit like I did here (in fact they should coincide + the PR merged after that one...)

https://github.com/cta-observatory/protopipe/blob/master/docs/CHANGELOG.rst

it could be good to add to the release drafter some editing notes so who makes a new release doesn't forget to use the same format